default_app/main.ts

```diff
@@ -1,20 +1,31 @@
-'use strict'
-
-const { app, dialog } = require('electron')
+import { app, dialog } from 'electron'
+
+import * as fs from 'fs'
+import * as path from 'path'
+import * as url from 'url'
+
+type DefaultAppOptions = {
+  file: null | string;
+  noHelp: boolean;
+  version: boolean;
+  webdriver: boolean;
+  interactive: boolean;
+  abi: boolean;
+  modules: string[];
+}
```

Comment:
All these needs to be marked as optional otherwise they'll all required even if you just want to specify one option.

```typescript
type DefaultAppOptions = {
  file?: string;
  noHelp?: boolean;
  version?: boolean;
  webdriver?: boolean;
  interactive?: boolean;
  abi?: boolean;
  modules?: string[];
}
```